### PR TITLE
[17.0] [MIG] supplier_calendar: Migration to 17.0

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

File: supplier_calendar/models/res_partner.py [hunk 1]
```diff
@@ -31,8 +31,6 @@ def supplier_plan_days(self, date_from, delta):
         :return: datetime: resulting date.
         """
         self.ensure_one()
-        if isinstance(delta, float):
-            delta = round(delta)
```
Comment (HviorForgeFlow) [inline supplier_calendar/models/res_partner.py]: ping @LoisRForgeFlow

Comment (LoisRForgeFlow) [reply supplier_calendar/models/res_partner.py]: Thanks @HviorForgeFlow 

Those lines are on purpose and need to be there. (ref: https://github.com/OCA/purchase-workflow/pull/2255)

Please @bizzappdev do not remove it

Comment (bizzappdev) [reply supplier_calendar/models/res_partner.py]: I added it back. @LoisRForgeFlow

**Review by LoisRForgeFlow — APPROVED** [review]
Thanks!

**Review by PCatinean — APPROVED** [review]

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**